scene/resources/visual_shader.cpp

```diff
@@ -782,7 +782,7 @@ Error VisualShader::_write_node(Type type, StringBuilder &global_code, StringBui
 			} else if (in_type == VisualShaderNode::PORT_TYPE_VECTOR && out_type == VisualShaderNode::PORT_TYPE_SCALAR) {
 				inputs[i] = "vec3(" + src_var + ")";
 			} else if (in_type == VisualShaderNode::PORT_TYPE_BOOLEAN && out_type == VisualShaderNode::PORT_TYPE_VECTOR) {
-				inputs[i] = "all(" + src_var + ")";
+				inputs[i] = "all(bvec3(" + src_var + "))";
```

Comment:
Is this an unrelated bugfix? I want to pull this into 3.1.x but I'm not sure about this.

Reply:
>  fixed conversion between vector and boolean.

It's safe to merge because it's not used anywhere before this PR